core/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/go-git/go-git/v5 v5.11.0
 	github.com/golang/mock v1.6.0
 	github.com/hashicorp/go-retryablehttp v0.7.5
+	github.com/natefinch/atomic v1.0.1
```

Comment:
why do you need this one? can you use the built-in atomic logic?

Reply:
Which built in atomic logic? According to the Go docs: "Even within the same directory, on non-Unix platforms os.Rename is not an atomic operation."

Reply:
I'll go ahead and take it out; we don't need to complicate the code to optimize for windows.